pkgs/development/interpreters/python/cpython/3.6/default.nix

```diff
@@ -64,6 +64,14 @@ in stdenv.mkDerivation {
     substituteInPlace configure --replace '-Wl,-stack_size,1000000' ' '
   '';
 
+  patches = optionals (!(stdenv.cc.isGNU or false)) [
```

Comment:
Well. The problem does not occur when using gcc, so I thought it would be best to make the intention clear. I'm not sure if applying this unconditionally might break something else, so I am basically being cautious.